media-sound/jack/jack-3.1.1-r2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="ncurses"
+inherit distutils-r1
+
+DESCRIPTION="A frontend for several cd-rippers and mp3 encoders"
+HOMEPAGE="http://www.home.unix-ag.org/arne/jack/"
+SRC_URI="http://www.home.unix-ag.org/arne/jack/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	dev-python/cddb-py[${PYTHON_USEDEP}]
```

Comment:
Please check whether the package builds successfully without those Python modules installed (i.e. unmerge it, then `emerge -1v --nodep media-sound/jack`. If not, they should land in DEPEND too.